charts/actions-runner-controller/crds/actions.summerwind.dev_runnerdeployments.yaml

```
@@ -982,6 +982,8 @@ spec:
                             - type: string
                           pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
                           x-kubernetes-int-or-string: true
+                        volumeUseMemory:
+                          type: boolean
```

```suggestion
                        volumeMedium:
                          type: string
```

Can we reuse the K8s API terminology here? Assuming all the Runner spec fields prefixed with `volume-` is for anything that is related to `/runner volume of the runner pod`, and reusing the K8s API's `StorageMedium`, this can be a good term.

https://sourcegraph.com/github.com/kubernetes/kubernetes/-/blob/pkg/apis/core/types.go?L633:2

---

Sure